applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -298,123 +281,101 @@ static void bt_codec_allocation_set(struct bt_codec_data *codec, enum bt_audio_l
-	struct bt_bap_broadcast_source_create_param create_param;
-
-	if (initialized) {
-		LOG_WRN("Already initialized");
-		return -EALREADY;
-	}
-
-	if (timestmp_cb == NULL) {
-		LOG_ERR("Timestamp callback is NULL");
-		return -EINVAL;
-	}
+	ARG_UNUSED(action);
 
-	timestamp_cb = timestmp_cb;
+	LOG_WRN("%s not supported", __func__);
+	return -ENOTSUP;
+}
 
-	(void)memset(audio_streams, 0, sizeof(audio_streams));
+int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay)
+{
+	ARG_UNUSED(bitrate);
+	ARG_UNUSED(sampling_rate);
+	ARG_UNUSED(pres_delay);
 
-	for (size_t i = 0; i < ARRAY_SIZE(stream_params); i++) {
-		stream_params[i].stream = &audio_streams[i];
-		bt_bap_stream_cb_register(stream_params[i].stream, &stream_ops);
-		stream_params[i].data_count = 1U;
-		stream_params[i].data = &bis_codec_data[i];
-		/* The channel allocation is set incrementally */
-		bt_codec_allocation_set(stream_params[i].data, BIT(i));
-	}
+	LOG_WRN("%s not supported", __func__);
+	return -ENOTSUP;
+}
 
-	for (size_t i = 0U; i < ARRAY_SIZE(subgroup_params); i++) {
-		subgroup_params[i].params_count = ARRAY_SIZE(stream_params);
-		subgroup_params[i].params = &stream_params[i];
-		subgroup_params[i].codec = &lc3_preset.codec;
-#if (CONFIG_BT_AUDIO_BROADCAST_IMMEDIATE_FLAG)
-		/* Immediate rendering flag */
-		subgroup_params[i].codec->meta[0].data.type = 0x09;
-		subgroup_params[i].codec->meta[0].data.data_len = 0;
-		subgroup_params[i].codec->meta_count = 1;
-#endif /* (CONFIG_BT_AUDIO_BROADCAST_IMMEDIATE_FLAG) */
-	}
+void le_audio_conn_set(struct bt_conn *conn)
+{
+	ARG_UNUSED(conn);
 
-	create_param.params_count = ARRAY_SIZE(subgroup_params);
-	create_param.params = subgroup_params;
-	create_param.qos = &lc3_preset.qos;
+	LOG_WRN("%s not supported", __func__);
+}
 
-	if (IS_ENABLED(CONFIG_BT_AUDIO_PACKING_INTERLEAVED)) {
-		create_param.packing = BT_ISO_PACKING_INTERLEAVED;
-	} else {
-		create_param.packing = BT_ISO_PACKING_SEQUENTIAL;
-	}
+int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_id)
+{
+	ARG_UNUSED(pa_sync);
+	ARG_UNUSED(broadcast_id);
 
-	if (IS_ENABLED(CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED)) {
-		create_param.encryption = true;
-		memset(create_param.broadcast_code, 0, sizeof(create_param.broadcast_code));
-		memcpy(create_param.broadcast_code, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY,
-		       MIN(sizeof(CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY),
-			   sizeof(create_param.broadcast_code)));
-	} else {
-		create_param.encryption = false;
-	}
+	LOG_WRN("%s not supported", __func__);
+	return -ENOTSUP;
+}
 
-	LOG_DBG("Creating broadcast source");
+void le_audio_conn_disconnected(struct bt_conn *conn)
+{
+	ARG_UNUSED(conn);
 
-	ret = bt_bap_broadcast_source_create(&create_param, &broadcast_source);
+	LOG_WRN("%s not supported", __func__);
+}
 
-	if (ret) {
-		LOG_ERR("Failed to create broadcast source, ret: %d", ret);
-		return ret;
-	}
+int le_audio_ext_adv_set(struct bt_le_ext_adv *ext_adv)
```

Comment:
A lot of unsupported functions. Can we remove them?

Reply:
That will be removed when we rewrite to bt_stream. The reason they are there is because they are all sharing one le_audio.h